fstapi/build.rs

```diff
@@ -1,26 +1,70 @@
-use std::{env, path::PathBuf};
+use std::{env, path::{self, PathBuf}};
+
+#[cfg(windows)]
+use vcpkg;
 
 fn main() {
+
+  #[cfg(windows)]
+  let (zlib_include_dir, mman_include_dir) = {
+    // Find zlib via vcpkg.
+    let zlib = vcpkg::Config::new()
+      .emit_includes(true)
+      .find_package("zlib")
+      .unwrap();
+    let zlib_include_dir = path::PathBuf::from(zlib.include_paths[0].clone());
```

Comment:
```suggestion
    let zlib_include_dir = PathBuf::from(zlib.include_paths[0].clone());
```

Use `PathBuf` directly.